pkgs/development/python-modules/htmllistparse/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, beautifulsoup4
+, html5lib
+, requests
+, fusepy
+}:
+
+buildPythonPackage rec {
+  pname = "htmllistparse";
+  version = "0.6.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-bcimvwPIQ7nTJYQ6JqI1GnlbVzzZKiybgnFiEBnGQII=";
```

```suggestion
    hash = "sha256-bcimvwPIQ7nTJYQ6JqI1GnlbVzzZKiybgnFiEBnGQII=";
```
https://nixos.org/manual/nixpkgs/unstable/#fetchurl